internal/cli/cli.go

```diff
@@ -119,6 +121,63 @@ func getReference(config Config, client resty.Client) (Reference, error) {
 
 }
 
+type TDPosition struct {
+	AveragePrice float64 `json:"averagePrice"`
+	Instrument struct {
+		AssetType string `json:"assetType"`
+		Cusip     string `json:"cusip"`
+		Symbol    string `json:"symbol"`
+	} `json:"instrument"`
+	LongQuantity float64 `json:"longQuantity"`
+}
+
+func (p *TDPosition) ToLot() Lot {
+	return Lot{
+		Symbol: p.Instrument.Symbol,
+		UnitCost: p.AveragePrice / p.LongQuantity,
```

Comment:
Looks like `averagePrice` is already the per-share cost.
```suggestion
		UnitCost: p.AveragePrice,
```

Reply:
Good catch. Done.